.github/container/install-nsight.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ex -o pipefail
+
+# Repo for newer nsight versions
+export UBUNTU_ARCH=$(dpkg --print-architecture)
+export UBUNTU_VERSION=$(. /etc/os-release && echo ${ID}${VERSION_ID/./}) # e.g. ubuntu2204
+curl -o /usr/share/keyrings/nvidia.pub https://developer.download.nvidia.com/devtools/repos/${UBUNTU_VERSION}/${UBUNTU_ARCH}/nvidia.pub
```

Comment:
Nitpick: the `https://developer.download.nvidia.com/devtools/repos/${UBUNTU_VERSION}/${UBUNTU_ARCH}` thing gets repeated twice here. Maybe use a variable to hold it and reuse?

Reply:
Changed.